react/features/base/toolbox/components/AbstractButton.js

```diff
@@ -255,13 +255,28 @@ export default class AbstractButton<P: Props, S: *> extends Component<P, S> {
         afterClick && afterClick();
     }
 
+    /**
+     * Helper function to be implemented by subclasses, which must return a
+     * {@code boolean} value indicating if this button should be rendered or not.
+     *
+     * @protected
+     * @returns {boolean}
+     */
+    _shouldRender() {
```

Comment:
The `visible` prop in `AbstractToolboxItem` does this already.

Reply:
Oh, right, I didn't realized there is one. Thanks!

Reply:
Yep, that works perfectly, thanks for the suggestion!